openwisp_users/multitenancy.py

```diff
@@ -118,6 +120,13 @@ def multitenant_behaviour_for_user_admin(self, request):
             qs = super().get_queryset(request)
         return qs
 
+    def formfield_for_foreignkey(self, db_field, request, **kwargs):
+        if db_field.name == 'organization':
+            kwargs['widget'] = OrganizationAutocompleteSelect(
+                db_field, self.admin_site, using=kwargs.get('using')
+            )
```

Comment:
Something I was forgetting, how does this change play out with changes like this one?

https://github.com/openwisp/openwisp-firmware-upgrader/commit/ae64e7f3278e3f5cf07984c072644d2f66fc2bf5#diff-495f69374c163b98300e8e981a9a2bea26f703c3dc960cbfb6a3c08ae1f7b0a1R56

A first look seem not to conflict with it, I guess we have to stop introducing that kind of change (adding organization to the autocomplete fields) in the different modules?

Reply:
Yes, I designed this solution in a way that will not require updating other modules of openwisp. Moreover, the default autocomplete fields added by Django won't have "Shared systemwide" option. We should not use it.